### PR TITLE
Dev docs cleanup and revision

### PR DESCRIPTION
**Pull request by tw2113**
Part of ongoing efforts for https://github.com/pfefferle/wordpress-webmention/issues/229

### CODE REVIEW

**Comment by pfefferle** [comment]
Nice, thanks!

File: includes/class-webmention-sender.php [hunk 1]
```diff
@@ -63,22 +62,22 @@ public static function comment_post( $id ) {
...
-	 * @param int $post_id the post_ID (optional)
+	 * @param string $source source url.
+	 * @param string $target target url.
+	 * @param int    $post_id the post_ID (optional).
 	 *
-	 * @return array of results including HTTP headers
+	 * @return array of results including HTTP headers.
 	 */
 	public static function send_webmention( $source, $target, $post_id = null ) {
 		// stop selfpings on the same URL
```
Comment (pfefferle) [inline includes/class-webmention-sender.php]: Shouldn't this be also start uppercase?

File: webmention.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 defined( 'WEBMENTION_VOUCH' ) || define( 'WEBMENTION_VOUCH', false );
 
-// initialize admin settings
+// initialize admin settings.
```
Comment (pfefferle) [inline webmention.php]: Capital "I" ?

File: webmention.php [hunk 3]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
...
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-sender.php';
 	add_action( 'init', array( 'Webmention_Sender', 'init' ) );
 
-	// initialize Webmention Receiver
+	// initialize Webmention Receiver.
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-receiver.php';
 	add_action( 'init', array( 'Webmention_Receiver', 'init' ) );
 
-	// initialize Webmention Vouch
+	// initialize Webmention Vouch.
```
Comment (pfefferle) [inline webmention.php]: Capital "I"?

File: webmention.php [hunk 4]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
...
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-410.php';
 	add_action( 'init', array( 'Webmention_410', 'init' ) );
 
-	// initialize Webmention Sender
+	// initialize Webmention Sender.
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-sender.php';
 	add_action( 'init', array( 'Webmention_Sender', 'init' ) );
 
-	// initialize Webmention Receiver
+	// initialize Webmention Receiver.
```
Comment (pfefferle) [inline webmention.php]: Capital "I"?

File: webmention.php [hunk 5]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
...
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-avatar-handler.php';
 	add_action( 'init', array( 'Webmention_Avatar_Handler', 'init' ) );
 
-	// load HTTP 410 support
+	// load HTTP 410 support.
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-410.php';
 	add_action( 'init', array( 'Webmention_410', 'init' ) );
 
-	// initialize Webmention Sender
+	// initialize Webmention Sender.
```
Comment (pfefferle) [inline webmention.php]: Capital "I"?

File: webmention.php [hunk 6]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
...
+	// list of various public helper functions.
 	require_once dirname( __FILE__ ) . '/includes/functions.php';
 
-	// load local avatar support
+	// load local avatar support.
 	require_once dirname( __FILE__ ) . '/includes/class-webmention-avatar-handler.php';
 	add_action( 'init', array( 'Webmention_Avatar_Handler', 'init' ) );
 
-	// load HTTP 410 support
+	// load HTTP 410 support.
```
Comment (pfefferle) [inline webmention.php]: Capital "L"?

File: webmention.php [hunk 7]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
 		require_once dirname( __FILE__ ) . '/includes/debug.php';
 	}
 
-	// list of various public helper functions
+	// list of various public helper functions.
 	require_once dirname( __FILE__ ) . '/includes/functions.php';
 
-	// load local avatar support
+	// load local avatar support.
```
Comment (pfefferle) [inline webmention.php]: Capital "L"?

File: webmention.php [hunk 8]
```diff
@@ -44,36 +44,36 @@ function webmention_init() {
 		require_once dirname( __FILE__ ) . '/includes/debug.php';
 	}
 
-	// list of various public helper functions
+	// list of various public helper functions.
```
Comment (pfefferle) [inline webmention.php]: Capital "L"?

File: includes/class-webmention-sender.php (hunk 1 above)
Comment (pfefferle) [reply includes/class-webmention-sender.php]: I am just asking, because I am not a native speaker and I am curious, what you would prefer.

**Comment by tw2113** [comment]
Regarding the capitalization spots, my PHPCS wasn't complaining, I believe because they were inline comments and not docblocks. I can definitely get them in a future pass if desired. It was complaining about the lack of punctuation. :D

**Comment by pfefferle** [comment]
No problem, I just wanted to know if it was on purpose ;) Thanks!

**Pull request merged**